core/dropdowndiv.ts

```diff
@@ -212,12 +212,13 @@ export function showPositionedByBlock(
  * @param opt_secondaryYOffset Optional Y offset for above-block positioning.
  * @returns True if the menu rendered below block; false if above.
  */
-export function showPositionedByField(
-    field: Field, opt_onHide?: Function,
+export function showPositionedByField<T>(
+    field: Field<T>, opt_onHide?: Function,
```

Comment:
Every other function in this namespace that's (1) exported, (2) not marked `@internal` and (3) references `Field` should also be updated to be generic. E.g. `showPositionedByBlock` above.

Reply:
Done!